python-threatexchange/threatexchange/exchanges/fetch_state.py

```diff
@@ -74,15 +74,16 @@ class SignalOpinionCategory(IntEnum):
 
     Some APIs may not support all of these, but each of these should influence
     what action you might take as a result of matching, otherwise it might
-    make more sense as a tag
+    make more sense as a tag.
     """
 
-    # TODO: Rename to DOES_NOT_FIT_CATEORY
-    FALSE_POSITIVE = 0  # Signal generates false positives
-    # TODO: Rename to something else
-    WORTH_INVESTIGATING = 1  # Indirect indicator
-    # TODO: Rename to FITS_CATEGORY
-    TRUE_POSITIVE = 2  # Confirmed meets category
+    # Negative class
+    # Signal generates false positives, or is test/example data
+    DOES_NOT_FIT_CATEGORY = 0
+    # Indirect indicator, requires verifying
+    MAY_FIT_CATEGORY = 1
+    # Positive class
+    FITS_CATEGORY = 2
```

Comment:
After sleeping on it, I propose:
```suggestion
    NEGATIVE_CLASS = 0
    # Indirect indicator, requires verifying
    INVESTIGATION_SEED = 1
    # Positive class
    POSITIVE_CLASS= 2
    # Maybe in the future: Informational 
```